engine/src/main/scala/cromwell/engine/WorkflowDescriptor.scala

```diff
@@ -40,6 +40,7 @@ case class WorkflowDescriptor(id: WorkflowId,
                               backend: Backend,
                               configCallCaching: Boolean,
                               lookupDockerHash: Boolean,
+                              configWorkflowFailureMode: Boolean,
```

Comment:
How about a proper type for this?  Not just because types are teh awesome but also as your title suggests it's not a stretch to imagine a "fail fastest" that aborts whatever's still running.